.github/workflows/tmate_debug/action.yml

```diff
@@ -9,6 +9,7 @@ runs:
   steps:
     - name: consider debugging
       shell: bash --noprofile --norc -eo pipefail -x {0}
+      if: runner.debug || contains(github.event.pull_request.labels.*.name, 'debug-ci')
```

Comment:
So we need to add a label to the PR `debug` if we need to test the CI?

And I would also say add a condition of when the test fails too, so we would not consider adding a label then

Reply:
these checks are added before the tests are started

Reply:
Even if the test fails, the new proposal is that we don't enable tmate by default because we so rarely will connect to it.